src/systemd/sd-lldp.h

```diff
@@ -129,6 +129,7 @@ int sd_lldp_stop(sd_lldp *lldp);
 
 int sd_lldp_attach_event(sd_lldp *lldp, sd_event *event, int64_t priority);
 int sd_lldp_detach_event(sd_lldp *lldp);
+sd_event *sd_lldp_get_event(sd_lldp *lldp);
```

Comment:
Well, lldp is not a structure, it's really a "living" object. As such, I doubt we should never expose functions with sd_lldp being const, after all this would otherwise declare that we'll never change the object's state internally. But that's an awfullly strong promise to make. For example, sd_lldp_get_event() could initialize something lazily, maybe lldp->event itself, or something completely unrelated (like for example some error code cache, or whatever else). Hence: I am pretty sure we should never add "const" to the objects in public APIs unless we really really really are sure that we'll never access the object for writing, and that means not even for caching something...